kwok/usersignup.go

```diff
@@ -0,0 +1,30 @@
+package kwok
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func setCluster(name, server string) error {
```

Comment:
are those used only for the tests, or with out code? if the former, might consider include them as fixtures in the test file.